### PR TITLE
first version

### PR DESCRIPTION
**Pull request by lcylcy**

### CODE REVIEW

**Comment by BBuf** [comment]
pr名字改一下

**Comment by doombeaker** [comment]
这个PR后续，交给 @EsdeathYZH 收尾

**Comment by doombeaker** [comment]
这个PR后续，交给 @EsdeathYZH 收尾

File: python/oneflow/test/modules/test_normalize.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import unittest
+from oneflow.test_utils.automated_test_util import *
+import oneflow as flow
```
Comment (doombeaker) [inline python/oneflow/test/modules/test_normalize.py:18]: 不必要导入

Comment (doombeaker) [inline python/oneflow/test/modules/test_normalize.py:18]: 不必要导入

File: python/oneflow/framework/docstr/normalize.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import oneflow
+from oneflow.framework.docstr.utils import add_docstr
+
+add_docstr(
+    oneflow.nn.functional.normalize,
+    r"""The interface is consistent with PyTorch.
```
Comment (doombeaker) [inline python/oneflow/framework/docstr/normalize.py:21]: 加一个函数原型，可以参考：https://github.com/Oneflow-Inc/OneTeam/blob/master/tutorial/howto_write_api_docs.md

Comment (doombeaker) [inline python/oneflow/framework/docstr/normalize.py:21]: 加一个函数原型，可以参考：https://github.com/Oneflow-Inc/OneTeam/blob/master/tutorial/howto_write_api_docs.md

**Comment by EsdeathYZH** [comment]
看了下，normalize这个op已经在 https://github.com/Oneflow-Inc/oneflow/pull/6940 实现了
